stable/kommander/values.yaml

```diff
@@ -42,3 +42,40 @@ ingress:
   # extraAnnotations:
   #   ingress.kubernetes.io/foo: bar
   path: /ops/portal/kommander
+
+federate:
+  addons: true
+  roles: true
+  rolebindings: true
+
+# This name must match the name of the kubefed dep
+# in order to override values.
+kubefed:
+  controllermanager:
```

Comment:
Do we need to override all these values? Looking at https://github.com/kubernetes-sigs/kubefed/blob/v0.1.0-rc6/charts/kubefed/README.md#configuration, the only non-default I can see are the `featureGates` (if you agree we disable everything other than `PushReconciler`)?

Reply:
i agree we should not override.

Reply:
Should not? ... Anyway, the reason for overriding in cases is to expose these values to kubeaddons in the future, if we so wish. For the featureGates we have to override as there is a problem with how null results are interpreted in kubefed. I will override these falues and leave only pushreconciler to enabled, but the rest have to be set to false explicitly.